sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveUtils.scala

```diff
@@ -207,8 +211,6 @@ private[spark] object HiveUtils extends Logging {
       ConfVars.METASTORE_AGGREGATE_STATS_CACHE_MAX_READER_WAIT -> TimeUnit.MILLISECONDS,
       ConfVars.HIVES_AUTO_PROGRESS_TIMEOUT -> TimeUnit.SECONDS,
       ConfVars.HIVE_LOG_INCREMENTAL_PLAN_PROGRESS_INTERVAL -> TimeUnit.MILLISECONDS,
-      ConfVars.HIVE_STATS_JDBC_TIMEOUT -> TimeUnit.SECONDS,
-      ConfVars.HIVE_STATS_RETRIES_WAIT -> TimeUnit.MILLISECONDS,
```

Comment:
What are the reasons we remove them?

Reply:
These two configurations were removed by [HIVE-12164](https://issues.apache.org/jira/browse/HIVE-12164).

Reply:
I do not know who is using it, but could we keep them? The constants are not available, but we can hard-code the names? Also add the code comments to explain the reasons why you change them?

Reply:
I created a new PR to fix this issue.: https://github.com/apache/spark/pull/24277